Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Melee/changeling_armblade.yml

```
@@ -10,13 +10,18 @@
       types:
         Slash: 15
         Structural: 20
-    clickPartDamageMultiplier: 2
-    heavyPartDamageMultiplier: 1.5
+    clickPartDamageMultiplier: 3
+    heavyPartDamageMultiplier: 2.5
+    heavyRangeModifier: 2.75
+    heavyRateModifier: 2.5
+    heavyDamageBaseModifier: 1.2
+    maxTargets: 1
+    angle: 10
```

YAMLlint (1.35.1)

[error] duplication of key "angle" in mapping

(key-duplicates)